packages/opentelemetry-tracing/src/Tracer.ts

```diff
@@ -50,7 +51,11 @@ export class Tracer implements api.Tracer {
   ) {
     const localConfig = mergeConfig(config);
     this._defaultAttributes = localConfig.defaultAttributes;
-    this._sampler = localConfig.sampler;
+    this._sampler =
+      config.sampler ||
+      new ProbabilitySampler(
+        Number(process.env.OTEL_SAMPLING_PROBABILITY || 1)
```

Comment:
Is this variable somewhere defined in documentation ?

Reply:
note to self: refer to https://github.com/open-telemetry/opentelemetry-collector/tree/master/processor#sampling-processor as an alternative to configuring sampling rate at service level.

Reply:
✅